### PR TITLE
Add syntax highlighting to the README

### PR DESCRIPTION
**Pull request by matan-h**
This would add syntax highlighting to the README, and the zsh shell case in the "Check buildozer is in your path" section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,74 +110,69 @@ buildozer android debug deploy
...
-    run        Run the application on the device
-    serve      Serve the bin directory via SimpleHTTPServer
+  distclean          Clean the whole Buildozer environment
+  help               Show the Buildozer help
+  init               Create an initial buildozer.spec in the current directory
+  serve              Serve the bin directory via SimpleHTTPServer
+  setdefault         Set the default command to run when no arguments are given
+  version            Show the Buildozer version
+
+Target commands: clean      Clean the target environment
```
Comment (misl6) [inline README.md:131]: I guess `clean         Clean the target environment` was expected to be on a new line. I'm right?

Comment (matan-h) [reply README.md]: Yes, I fixed this

**Comment by Julian-O** [comment]
@matan-h: I have submitted a PR 1750 (see link above) with a major overhaul to the ReadMe. 

I think it renders this PR obsolete.

If you would like to review it to make sure it covers all the changes you made, I would appreciate it.

Then we can close this one.

**Comment by matan-h** [comment]
I Agree. @Julian-O pr (#1750) is better.

**Pull request closed without merging**